src/main/scala/org/antipathy/scoozie/action/HasConfig.scala

```diff
@@ -0,0 +1,30 @@
+// $COVERAGE-OFF$
+package org.antipathy.scoozie.action
+
+import org.antipathy.scoozie.configuration.Configuration
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use brackets to group imports from the same package](https://app.codacy.com/app/SimonJPegg/scoozie/pullRequest?prid=3078561)